ziti/ziti.go

```diff
@@ -724,10 +724,13 @@ func (mgr *listenerManager) handleRouterConnectResult(result *edgeRouterConnResu
 			mgr.routerConnections[routerConnection.GetRouterName()] = routerConnection
 			go mgr.createListener(routerConnection, mgr.session)
 		}
+	} else {
+		pfxlog.Logger().Debugf("ignoring connection to %v, already have max connections %v", result.routerUrl, len(mgr.routerConnections))
```

Comment:
seems like a warning to me vs debug? just judging by the message this 'sounds' like a bad thing and i'd want to see that in the log?

Reply:
It's not a bad thing. The way it works as follows:

1. There's a configured number of max connections when hosting a service
2. When starting to host a service it will create a bind session. The bind session returns all the edge routers available to host the service.
3. The SDK will attempt to connect to all available edge routers simultaneously. As connections are made we'll try to listen over them, but only up the max number of configured connections. If we have 5 available edge routers and 3 max connections, we'll connect to the first three that come back. The next two will hit this condition, but it's fine.